packages/schematics/package-update/_angular/index.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { Rule } from '@angular-devkit/schematics';
+import { SchematicsUpdateSchema } from '../schema';
+import { updatePackageJson } from '../utility/npm';
+
+
+const angularPackagesName = [
```

Comment:
This list shouldn't have any package names with 3+ parts, like `@*/*/testing` or `@angular/platform-browser/animations`. Those aren't real package names.

Reply:
Removed.